src/components/commons/HeroBanner.tsx

```diff
@@ -0,0 +1,103 @@
+import { Container } from "@components/commons/Container";
+import classNames from "classnames";
+import { SectionTitle } from "@components/commons/SectionTitle";
+import { Button } from "@components/commons/Buttons";
+import { StartExploring } from "@components/commons/StartExploring";
+
+// The HeroBanner component takes in title, subtitle, desc and HeroBannerBG
+// currently only COIN_ARROW, DFI_COIN and CUBE image are available
+// it also takes in optional params on the customBgPosStyle
+// if cta button is needed to pass in ctaButton, text and link
+// if start exploring button is needed to pass in start exploring and jumplink
+export function HeroBanner({
+  title,
+  subtitle,
+  desc,
+  customBgPosStyle,
+  heroBG,
+  ctaButton = false,
+  ctaButtonLink,
+  ctaButtonText,
+  startExploring = false,
```

Comment:
#nitpick, we generally want to name variables as close to its type as possible
e.g.: if it's a flag, then maybe prefix with `is`, `has`, `can`

go through the list, and see if you'll know what type are they on first glance without checking type

Reply:
updated to clearer flag names and desc !